src/egon/data/datasets/industrial_sites/__init__.py

```diff
@@ -689,7 +680,7 @@ class MergeIndustrialSites(Dataset):
     def __init__(self, dependencies):
         super().__init__(
             name="Merge_industrial_sites",
-            version="0.0.1",
+            version="0.0.2",
```

Comment:
The version number was already updated to 0.0.2 on the CI-branch. Could you please update the version number to 0.0.3 to be sure it is also updated on the CI-branch?